index.html

```diff
@@ -285,7 +285,7 @@ <h3 class="Skills-title">Skills</h3>
 
       <!-- Form start  -->
 
-      <form>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ optional ] Good job for using form-spree for for submission, But when you are using [formspree service](https://formspree.io/html) you have to create your own hash id  <form action="https://formspree.io/f/{your_id}" method="post">. When I submitted it show the below image,
![Capture](https://user-images.githubusercontent.com/93598189/192610092-19162dce-0d96-4f84-bf79-3737cb0b0c90.PNG)

Reply:
check the bellow commit

Reply:
Hello @Mujeeb4582 Good Job 👍

Reply:
check the latest commit

Reply:
@Abenezer-Tilahun

Reply:
I checked it and it done well , I approved the Project and go head and merge it.

Reply:
thank you sir